src/CMakeLists.txt

```diff
@@ -120,11 +120,7 @@ set(ext_header_files
 # Base target for common options.
 add_library(_lexy_base INTERFACE)
 target_sources(_lexy_base INTERFACE ${header_files})
-if ("cxx_std_20" IN_LIST CMAKE_CXX_COMPILE_FEATURES)
```

Comment:
Based on the discussion in #55, please make the proposed behavior conditional by introducing a CMake option `LEXY_FORCE_CPP17`.

Reply:
To clarify, `LEXY_FORCE_CPP17=ON` means to use `target_compile_options(_lexy_base INTERFACE cxx_std_17)`, and in the `OFF` case, to use the old behaviour. Is this correct?

Reply:
Yes, and the option is off by default.